ddp-workspace/projects/ddp-pancan/src/app/components/page-with-sections/page-with-sections.component.scss

```diff
@@ -151,33 +184,55 @@
 }
 
 @media screen and (max-width: $desktop-breakpoint) {
-    .section__side-image {
-        .section-0 & {
-            height: 480px;
-            margin-right: 5rem;
+    .section {
+        &-0 {
+            background-size: auto 480px;
         }
 
-        .section-1 & {
```

Comment:
#minor btw google style guide recommends use full classes like '.section-0' instead '&-0' in order to allow developers find it in the css code easier